src/test/java/net/starlark/java/eval/Benchmarks.java

```diff
@@ -61,15 +62,16 @@
 public final class Benchmarks {
 
   private static final String HELP =
-      "Usage: Benchmarks [--help] [--filter regex] [--seconds float]\n"
+      "Usage: Benchmarks [--help] [--filter regex] [--seconds float] [--iterations count]\n"
           + "Runs Starlark benchmarks matching the filter for the specified (approximate) time,\n"
           + "and reports various performance measures.";
 
   private static boolean ok = true;
 
   public static void main(String[] args) throws Exception {
     Pattern filter = null; // default: all
-    long budgetNanos = 1_000_000_000;
+    long budgetNanos = 0;
```

Comment:
These sentinel values should be negative so that --iterations=0 does the right thing (useless though it is).